docs/release-notes/8.2.asciidoc

```diff
@@ -2,14 +2,27 @@
 == 8.2
 
 [discrete]
-[[known-issue-8.2.0]]
-==== Known issues
-* On macOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs), users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4 or later.
+[[release-notes-8.2.1]]
+=== 8.2.1
+
+[discrete]
+[[bug-fixes-8.2.1]]
+==== Bug fixes and enhancements
+* Allows {kibana-ref}/pre-configured-connectors.html[preconfigured connectors] to be used with cases ({pull}130372[#130372]).
+* Adds pagination to the *Table* tab on the Alert details flyout to fix a performance issue on the Timelines page ({pull}131358[#131358]).
+* Fixes sorting issues that were related to unmapped fields ({pull}132190[#132190]).
+* Fixes a bug in the *Filter In*, *Filter Out*, and *Add to timeline investigation* inline actions that caused incorrect results to be retrieved ({pull}132251[#132251]).
+* Calculates the top count function and hover action in a more performant way ({pull}131363[#131363]). 
```

Comment:
Minor quibble; "in a more performant way" feels a little awkward to me:
```suggestion
* Improves performance of calculating the top count function and hover action ({pull}131363[#131363]). 
```

Reply:
Hm, so this PR actually changes how top count results are calculated and makes the whole process more efficient and less resource-intensive. If I removed "performance of" from your edit, the summary would still be correct...but it'd obscure some details about what was improved. How about:

Enhances performance by improving calculations for the top count function and hover action.

@kqualters-elastic WDYT?